libraries/chain/include/eos/chain/types.hpp

```diff
@@ -171,7 +171,7 @@ namespace eos { namespace chain {
       transaction_object_type,
       producer_object_type,
       chain_property_object_type,
-      transaction_history_object_type, ///< Defined by account history plugin library
+      transaction_history_object_type, ///< Defined by account_history_plugin library
```

Comment:
I apologize, I have been unclear. This comment claims the object is defined by a plugin library, but it's not a library, it's a plugin. The purpose of the comment is to make it easier to find the object without an IDE tracking references of `transaction_history_object_type`, so if it says "library" and "plugin", it's not obvious whether to look in the `plugins` or `libraries` directory. This comes from a plugin, so it should only say plugin and not library.